src/css/doc.css

```diff
@@ -1261,14 +1261,12 @@ body {
   margin: 0 0.2rem;
   content: "→";
   color: var(--link-font-color);
-  transform: rotate(-45deg);
   transition: all 0.2s linear;
   top: 0;
 }
 
 .doc a.external:hover::after {
-  top: -12px;
-  opacity: 0;
+  text-decoration: underline;
```

Comment:
I would remove this. Links are already underlined, so adding this `text-decoration` just adds an underline to the arrow when hovering over an external link. (This change applies to all external links in the docs)

Reply:
Yeah, I don't really like that animation. The two lines that are deleted here I'm ok with that change. I just wouldn't add the text-decoration here.